src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-workflows/migration-to-workflows.mdx

```diff
@@ -0,0 +1,627 @@
+
+## What to expect [#what-to-expect]
+
+The migration from the classic notification system to workflows creates a **destination** for each **classic notification channel** and connects them to **workflows** created for each **policy.** Only policies with channel associations will be migrated.
+
+1. New Relic will automatically migrate accounts from January 9-31 2023.
+2. New Relic can migrate accounts earlier, just reach out to your account team.
+3. Your team can avoid the automated migration by removing channels from policies.
+4. Notifications will not substantially change, specify which values and for which notification channel types.
+
+## Configuring custom payloads [#configure-custom-payloads]
+
+When moving from notification channels to workflows your team may want to make some tweaks to your custom payloads. Workflows still function in the same way as notifications in that, when a condition is violated then a notification is sent to a webhook and when it is sent, it goes with its custom payload. The migration from notification channels to workflows requires changing some of the terminology in this payload.
+
+The following table provides a translation between what the webhook payload names used in our classic notification system and their new, corresponding names in the issue payload.  
+
+For many keys, the issue payload may contain a list of values.
+
+<table>
+
+  <thead>
+    <tr>
+      <th>
+        **Alerts (classic) name**
+      </th>
+
+      <th>
+        **Alerts (classic) variable**
+      </th>
+
+      <th>
+        **Workflow template replacement**
+      </th>
+
+      <th>
+        **Notes**
+      </th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <tr>
+      <td>
+        `account_id`
+      </td>
+      <td>
+        `$ACCOUNT_ID`
+      </td>
+      <td>
+        `{{nrAccountId}}`
+      </td>
+      <td>
+        The account associated with the issue. 
+      </td>
+    </tr>
+
+
+    <tr>
+      <td>
+        `account_name`
+      </td>
+      <td>
+        `$ACCOUNT_NAME`
+      </td>
+      <td>
+        `{{ accumulations.tag.account.[0] }}`
+      </td>
+      <td>
+        An issue can have multiple accounts.
+      </td>
+    </tr>
+
+
+      <tr>
+      <td>
+        `closed_violations_count_critical`
+      </td>
+      <td>
+        `$CLOSED_VIOLATIONS_COUNT_CRITICAL`
+      </td>
+      <td>
+        `{{closedIncidentsCount}}`
+      </td>
+      <td>
+        Closed incident counts of all incidents regardless of priority.
+      </td>
+    </tr>
+
+
+    <tr>
+      <td>
+        `closed_violations_count_warning`
+      </td>
+      <td>
+        `$CLOSED_VIOLATIONS_COUNT_WARNING`
+      </td>
+      <td>
+        N/A
+      </td>
+      <td>
+        There is no replacement for warning counts.  All open incident counts will be represented as critical to avoid double counting incidents. 
```

Comment:
```suggestion
        There is no replacement for warning counts.  All closed incident counts will be represented as critical to avoid double counting incidents. 
```